spec/features/idv/steps/in_person/ssn_spec.rb

```diff
@@ -111,7 +111,7 @@
       # location page
       complete_location_step(user)
       # state ID page
-      fill_out_state_id_form_ok(double_address_verification: true, same_address_as_id: false)
+      fill_out_state_id_form_ok(same_address_as_id: false, capture_secondary_id_enabled: true)
       click_idv_continue
       fill_out_address_form_ok(double_address_verification: true, same_address_as_id: false)
```

Comment:
The usage of both `double_address_verification` and `capture_secondary_id_enabled` here seems likely to cause confusion. I think we should only use one or the other for the test methods in general unless a specific test needs to differentiate.

My preferred approach in this case is reverting to `double_address_verification` for most usages & mapping to `capture_secondary_id_enabled` where needed.

Reply:
i'm going to go ahead and merge since we'll be removing `dav` though I understand it might be confusing to see the different params for `state_id_form` vs `address_form`